horreum-client/src/test/java/io/hyperfoil/tools/HorreumTestBase.java

```diff
@@ -0,0 +1,219 @@
+package io.hyperfoil.tools;
+
+import io.hyperfoil.tools.horreum.entity.json.Test;
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.ContainerState;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.output.OutputFrame;
+import org.testcontainers.containers.output.WaitingConsumer;
+import org.testcontainers.utility.LogUtils;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+
+public class HorreumTestBase {
+
+    static Properties configProperties;
+    public static final String HORREUM_KEYCLOAK_BASE_URL;
+    private static final String HORREUM_BASE_URL;
+
+    public static final String HORREUM_USERNAME;
+    public static final String HORREUM_PASSWORD;
+    private static final String HORREUM_CLIENT_SECRET;
+    private static final Integer HORREUM_TEST_PORT_OFFSET;
+
+    private static final String CONTAINER_HOST_IP;
+
+    private static final Integer HORREUM_HTTP_PORT = 8082;
+    private static final Integer HORREUM_HTTPS_PORT = 8443;
+    private static final Integer GRAFANA_HTTP_PORT = 4040;
+    private static final Integer POSTGRES_PORT = 5432;
+    private static final Integer KEYCLOACK_PORT = 8180;
```

Comment:
You have a typo at multiple places: `KEYCLOACK` -> `KEYCLOAK`.

Reply:
Thats what happens when you copy/paste an incorrect spelling. Will fix